src/pages/publication/PublicationPage.tsx

```diff
@@ -73,7 +73,7 @@ const PublicationPage: FC<PublicationPageProps> = ({ publicationId }) => {
         <ContentPage>
           {publication ? (
             <>
-              <Heading>{publication.title?.no}</Heading>
+              <Heading>{publication.entityDescription.mainTitle}</Heading>
```

Comment:
hadde kanskje gjort seg med en destrukturering her også

Reply:
+1

Reply:
stemmer det, grunnen til at jeg ikke gjorde det var at `publication` kan være null som funker dårlig for deconstructing :/

Reply:
ser ut til å virke med slik destrukturering:
```
const { mainTitle, abstract, description, npiDiscipline, tags, date, language, projects } =
    (typeof publication !== 'undefined' && publication.entityDescription) || {};
```

ganske høgg, men jeg liker det faktisk bedre likevel

Reply:
eller så kan vi dele opp i flere returns basert på noen if-setninger

Reply:
smart 👍